charts/ingress-nginx/values.yaml

```diff
@@ -793,6 +793,15 @@ controller:
       # requests:
       #   cpu: 10m
       #   memory: 20Mi
+      extraVolumeMounts: []
+      ## Additional volumeMounts to the AdmissionsWebhook createSecretJob.
+      #  - name: kube-api-access
+      #    mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      #    readOnly: true
+      extraVolumes: []
+      ## Additional volume to the AdmissionsWebhook createSecretJob.
+      #  - name: kube-api-access
+      #    emptyDir: {}
```

Comment:
Please put the comment before the actual property and use `# --` notation for documentation.